app/(pages)/about-us/page.tsx

```diff
@@ -1,10 +1,107 @@
-import { Stack } from '@mui/joy';
+import { Stack, Box, Typography, Card, CardOverflow, Grid, AspectRatio, CardContent,Sheet } from '@mui/joy';
+import Image from 'next/image';
 import PageHeaderCard from '@/components/PageHeaderCard';
+import HeadingBodyText from '@/components/HeadingBodyText';
+import OutlineButton from '@/components/OutlineButton';
 
+interface cardProps {
+  picture: string;
+  cardName: string;
+  cardText: string;
+  link: string;
+}
+
+function AboutUsCard({picture, cardName, cardText,link}: cardProps) {
+  return (
+      <Card className='shadow-hover' variant='plain' sx={{ width: {xs: '90%'}}}>
+        <CardOverflow>
+          <AspectRatio ratio='2'>
+            <Image 
+            src={picture}
+            alt=''
+            fill
+            />
+          </AspectRatio>
+        </CardOverflow>
+
+        <CardContent>
+          <Stack direction='column' alignItems='center'>
+            <Typography level='h2'
+              textAlign='center'>
+              {cardName}
+            </Typography>
+            <Typography level='body' pb={3}
+              textAlign='center'>
+              {cardText}
+            </Typography>
+            <Box sx={{ mb: 3 }}> 
+            <OutlineButton text='Learn More' link={link} />
+          </Box>
+          </Stack>
+        </CardContent>
+      </Card>
+  );
+
+}
 export default function AboutUs() {
+
+  const aboutWIEsocText = 'We are a UNSW student-run society that facilitates members from all \n \
```

Comment:
Can you remove the newline \n characters? I'm putting in a style that renders them so it'll look funny haha. You can keep the '\'